src/test/scala/chisel3/experimental/tests/VcsBasicTests.scala

```diff
@@ -0,0 +1,134 @@
+package chisel3.experimental.tests
+
+import chisel3._
+import chisel3.tester._
+import chisel3.tester.experimental.TestOptionBuilder._
+import chisel3.tester.internal.VcsBackendAnnotation
+import chisel3.tests.{PassthroughModule, StaticModule}
+import org.scalatest._
+
+class VcsBasicTests extends FlatSpec with ChiselScalatestTester with Matchers {
+  behavior of "Testers2_Vcs"
+
+  val annos = Seq(VcsBackendAnnotation)
+
+  it should "test static circuits" in {
+    assume(firrtl.FileUtils.isVCSAvailable)
+
+    test(new StaticModule(42.U)).withAnnotations(annos) { c =>
+      c.out.expect(42.U)
+    }
+  }
+
+  it should "fail on poking outputs" in {
+    assume(firrtl.FileUtils.isVCSAvailable)
+
+    assertThrows[UnpokeableException] {
+      test(new StaticModule(42.U)).withAnnotations(annos) { c =>
+        c.out.poke(0.U)
+      }
+    }
+  }
+
+  it should "fail on expect mismatch" in {
+    assume(firrtl.FileUtils.isVCSAvailable)
+
+    assertThrows[exceptions.TestFailedException] {
+      test(new StaticModule(42.U)).withAnnotations(annos) { c =>
+        c.out.expect(0.U)
+      }
+    }
+  }
+
+  it should "fail with user-defined message" in {
```

Comment:
Ditto, is this VCS-specific? I don't think so, or at least the control paths are redundant with the fail-on-expect simple case?